head/simulator/main.cpp

```diff
@@ -133,10 +132,8 @@ static motor_interrupt_driver::MotorInterruptDriver sim_interrupt_left(
     motor_queue_left, motor_interrupt_left, motor_interface_left,
     update_position_queue_left);
 
-static auto adc_comms = adc::SimADC{};
-
 static auto presence_sense_driver =
-    presence_sensing_driver::PresenceSensingDriver(adc_comms);
+    presence_sensing_driver::PresenceSensingSimulator();
```

Comment:
Should we default this to all instruments being attached rather than unattached? Otherwise I think the hardware controller will not return that instruments are attached if using the `OneShotDetector` class (or whatever it was called).

Reply:
it'll asplode if it's wrong though i think